CONTRIBUTING.md

```diff
@@ -54,12 +49,7 @@ help prioritize the most common problems and requests.
 
 ### Conventions
 
-Fork the repo and make changes on your fork in a feature branch:
-
-- If it's a bugfix branch, name it XXX-something where XXX is the number of the
```

Comment:
No need for explicit branch names. Personally I have never looked at the branch name of a PR

Reply:
Maybe we should say that they at least should use a branch PR from the master give a lot of extra troubles with rebasing